release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -19,7 +19,11 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 {product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {cloud-redhat-com} application for {product-title} allows you to deploy OpenShift clusters to either on-premise or cloud environments.
 
 // Double check OP system versions
+<<<<<<< HEAD
 {product-title} {product-version} is supported on {op-system-base-full} 8.4, as well as on {op-system-first} 4.10.
+=======
+{product-title} {product-version} is supported on {op-system-base-full} 7.9 and 8.4, as well as on {op-system-first} 4.9.
+>>>>>>> 7bfffb698... TELCODOCS-261: Redfish hardware events
```

Comment:
@tmulquee Looks like you've got a bit of merge conflict residue.

Reply:
Hi Chris, thanks, I believe that should be clean now.